scripts/build-controller-image.sh

```diff
@@ -12,6 +12,20 @@ LOCAL_MODULES=${LOCAL_MODULES:-"false"}
 BUILD_DATE=$(date +%Y-%m-%dT%H:%M)
 QUIET=${QUIET:-"false"}
 
+HARDWARE_PLATFORM=$(uname -i)
```

Comment:
It would be awesome if we could make this configurable

Double check this for me, but I think something like this:
```suggestion
HARDWARE_PLATFORM=${HARDWARE_PLATFORM:-(uname -i)}
```